miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/RealMiniApp.kt

```diff
@@ -73,10 +75,37 @@ internal class RealMiniApp(
         }
     }
 
-    override fun clearSecureStorage() = secureStorageDispatcher.clearSecureStorage()
+    @Suppress("TooGenericExceptionCaught", "SwallowedException")
+    override fun clearSecureStorage(context: Context) {
```

Comment:
I think this could be named `clearSecureStorage**s**` to differentiate with `clearSecureStorage(context: Context, miniAppId: String)`

Reply:
@laubengaier addressed.